rfcs/2022-04-09-accessibility.md

```diff
@@ -0,0 +1,505 @@
+
+Other subcategories based on [microsoft’s inclusive design](https://www.microsoft.com/design/inclusive):  
+
+- **Permanent or disability that does not go away:**:   
+  - Example: Someone who is deaf
+- **Temporary, disability that will go away in time:**: 
+  - Example: Someone who has an ear infection
+- **Situational, Context based disabilities:**: 
+  - Example: Someone who is at a loud sporting event
+  - Example: Using the web app in bright sunlight
+  - Example: Using your phone one handedly in a crowded metro station
+
+<img src="./images/accessibility/ms-inclusive-design.png" alt="Microsoft Inclusive Design Shows Spectrum Of Disabilities" width="25%" />
+
+## Accessibility Principles
+
+According to [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) or WCAG, Web contents should be: 
+
+**Perceivable**
+
+- Provide text alternatives for non-text content.
+- Provide captions and other alternatives for multimedia.
+- Create content that can be presented in different ways, including by assistive technologies, without losing meaning.
+- Make it easier for users to see and hear content.
+
+**Operable**
+
+- Make all functionality available from a keyboard.
+- Give users enough time to read and use content.
+- Do not use content that causes seizures or physical reactions.
+- Help users navigate and find content.
+- Make it easier to use inputs other than keyboard.
+
+**Understandable**
+
+- Make text readable and understandable.
+- Make content appear and operate in predictable ways.
+- Help users avoid and correct mistakes.
+
+**Robust**
+
+- Maximize compatibility with current and future user tools.
+
+# Motivation
+- Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+## Why are we doing this? 
+
+Without proper accessibility, certain functionality used in web sites is not available to some users with disabilities, especially people who rely on screen readers and people who cannot use a mouse. 
+
+Making our products accessible is important for two reasons.  
+- it's the right thing to do. We believe that everyone deserves to be able to use our products, regardless of their abilities.
+- It'll help build trust with our customers. By making our products accessible, we can expand our potential customer base and make our products more appealing to a wider range of people. 
+
+> [The Business Case Study for Accessibility](https://www.w3.org/WAI/business-case/)
+
+**Making customers happy & respecting human rights:**  
+
+> In 2020, A 29 year old banker who is visually impaired opened petitions to make [zomato](https://www.change.org/p/zomato-make-zomato-accessible-with-screen-readers-for-visually-challenged-people) & [swiggy](https://www.change.org/p/swiggy-make-swiggy-accessible-with-screen-readers-in-android-for-visually-challenged-people) accessible because they couldn’t order food from it due to their inaccessible apps.  
```

Comment:
```suggestion
> In 2020, A 29 year old banker who is visually impaired opened petitions to make [Zomato](https://www.change.org/p/zomato-make-zomato-accessible-with-screen-readers-for-visually-challenged-people) & [Swiggy](https://www.change.org/p/swiggy-make-swiggy-accessible-with-screen-readers-in-android-for-visually-challenged-people) accessible because they couldn’t order food from it due to their inaccessible apps.  
```